plugins/node/opentelemetry-instrumentation-router/src/types.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Span } from '@opentelemetry/api';
+
+export enum LayerType {
+  MIDDLEWARE = 'middleware',
+  REQUEST_HANDLER = 'request_handler',
+}
+
+export enum CustomAttributeNames {
```

Comment:
for consistency with other packages, culd you move those custom attributes to `enums/AttributeNames.ts` ? https://github.com/open-telemetry/opentelemetry-js-contrib/commit/aec35d8a5e3d06c39db58c1ac7d20555163dc709

Reply:
Moved that under enums. Stuff's changing underneath me, hard to keep up. :)